site/content/contribute/mobile/build-your-own/environment-vars.md

```diff
@@ -9,6 +9,8 @@ In order to use the **make build**, **make build-android** and/or **make build-i
 
 * Copy the file with your variables to `./mattermost-mobile/fastlane/.env` where `.env` is the file name.
 
+Additionaly a sample file `env_vars_example` can be found on `./mattermost-mobile/fastlane/`
```

Comment:
```suggestion
Additionally, a sample file `env_vars_example` can be found on `./mattermost-mobile/fastlane/`.
```

Reply:
Hey @Selimix, would you be open to helping with this one change from Amy? We don't have permission to push to your branch.

Many thanks for your contribution!